### DISCUSSION TITLE
Delete custom converter

### DISCUSSION THREAD

**Discussion by Eguar-rc** [Koenkk/zigbee2mqtt#18822] [Devices]
I have removed a custom converter because the device is already officially included but the custom converter data still appears instead of the new ones, the device is the same so the IEEE is maintained.
How can the previous trace be completely removed?

**Comment by LemmeDasker** [comment]
Just an idea...
Did you shutdown zigbee2mqtt/mosquitto and unplugged the USB stick. And then startet it all again?

**Comment by Eguar-rc** [comment]
I had only rebooted Z2M after deleting the custom component, now I followed your suggestion and... it worked! 
Now the device appears with the official image and data, thanks

**Comment by LemmeDasker** [comment]
You are welcome :-)
